test/functional/tests/cli/test_cli_help_spelling.py

```diff
@@ -0,0 +1,230 @@
+#
+# Copyright(c) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+import pytest
+import time
+
+from api.cas import casadm
+from api.cas.cli import casadm_bin
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from test_utils.size import Unit, Size
+
+
+def test_cli_help_spelling():
+    """
+    title: Spelling test for 'help' command
+    description: Validates spelling of 'help' in CLI
+    pass criteria:
+    - no spelling mistakes are found
+    """
+
+    personal_directory = "/root/open-cas-linux/test/smoke_test/cas_ex.en.pws"
```

Comment:
I'm not sure whether we want to rely on an existing config file, especially if it is located somewhere in repo. I suggest converting this config file into a list within a test and spawning it on a DUT during each test execution.

Reply:
Moreover, we can not assume the repo location, because users can put their repo in any directory.